rotary/utils.py

```diff
@@ -100,3 +100,39 @@ def symlink_or_compress(in_file_path, out_file_path):
         os.symlink(in_file_path, out_file_path)
     else:
         gzip_file(in_file_path, out_file_path)
+
+
+def get_contamination_reference_files(config_entry: list, db_path: str, phix_accession: str, human_accession: str):
+    """
+    Returns a list of file paths for contamination references based on short IDs
+
+    :param config_entry: value of config.get('contamination_references'); should be a list
+    :param db_path: path to the rotary database folder
+    :param phix_accession: NCBI assembly accession code for the PhiX genome assembly
+    :param human_accession: NCBI assembly accession code for the human genome assembly
+    :return: list of file paths for the contamination reference sequences
+    """
+
+    contamination_reference_paths = []
+
+    if isinstance(config_entry, list) is True:
```

Comment:
Also, you can remove the is True here and it should still work.:

```python
isinstance(config_entry, list):
   do stuff..
```

I usually do this with functions that evaluate something for example functions that evaluate only true or false. I.e., is the config entry an instance of list? Yes or No.

Reply:
Good to know I can simplify this -- I was trying to keep the True/False test obvious in the code for readability, but I think you're right that the function name itself makes it pretty clear what the code is doing. Just addressed this.